packages/@aws-cdk/aws-iot-actions/test/kinesis-firehose/integ.firehose-put-record-action.expected.json

```diff
@@ -123,10 +123,6 @@
                 "s3:List*",
                 "s3:DeleteObject*",
                 "s3:PutObject",
-                "s3:PutObjectLegalHold",
-                "s3:PutObjectRetention",
-                "s3:PutObjectTagging",
-                "s3:PutObjectVersionTagging",
```

Comment:
Sorry, what happened here?

Reply:
Sorry, my local build is old... I've fix it!